tests/e2e/initDevfileTests.sh

```diff
@@ -9,19 +9,33 @@ launchAllUserstories(){
     echo "Launching all userstories";
     echo ""
 
-    npm run lint && npm run tsc && mocha --config mocha-all-devfiles.json ;
+    if [ "${TS_SELENIUM_EDITOR}" == "theia" ]; then
+        npm run lint && npm run tsc && mocha --config mocha-all-devfiles-theia.json ;
+    elif [ "${TS_SELENIUM_EDITOR}" == "code" ]; then
+        npm run lint && npm run tsc && mocha --config mocha-all-devfiles-code.json ;
+    fi
 }
 
 launchSingleUserstory(){
     echo ""
     echo "Launching the \"${USERSTORY}\" userstory";
     echo ""
 
-    tsc && mocha --config mocha-single-devfile.json --spec dist/tests/login/Login.spec.js --spec dist/tests/devfiles/${USERSTORY}.spec.js ;
+    if [ "${TS_SELENIUM_EDITOR}" == "theia" ]; then
+        tsc && mocha --config mocha-single-devfile.json --spec dist/tests/login/Login.spec.js --spec dist/tests/devfiles/theia/${USERSTORY}.spec.js ;
```

Comment:
It could be simplified:
```
tsc && mocha --config mocha-single-devfile.json --spec dist/tests/login/Login.spec.js --spec dist/tests/devfiles/${TS_SELENIUM_EDITOR}/${USERSTORY}.spec.js
```